src/test/java/com/rybka/service/export/JsonExportServiceTest.java

```diff
@@ -0,0 +1,107 @@
+package com.rybka.service.export;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.rybka.configuration.ExportProperty;
+import com.rybka.exception.ExportFailureException;
+import com.rybka.model.CurrencyHistory;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Calendar;
+import java.util.List;
+
+import static org.mockito.Mockito.*;
+
+@ExtendWith(SpringExtension.class)
+class JsonExportServiceTest {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    @Mock
+    private ExportProperty exportProperty;
+
+    @Test
+    public void testOnProperExport() {
+
+        // given
+        var testFolder = "src/test/resources/";
```

Comment:
The same comments as for the CsvExportServiceTest.

Reply:
fixed